### PR TITLE
Add "Gymnasium-MuJoCo-v5 Environments" Blog Post

### PR DESCRIPTION
**Pull request by Kallinteris-Andreas**
post was reviewed at: https://github.com/Farama-Foundation/Gymnasium/pull/945

Should be merged during 1.0 gymnasium release

### CODE REVIEW

File: _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Here all we have to do is change the `xml_file` argument, and set the `healthy_z_range`, because the robot has a different height than the default `Ant` robot.  In general, we will have to change the `healthy_z_range` to fit the robot.
+
+<iframe id="odysee-iframe" width="560" height="315" src="https://odysee.com/$/embed/@Kallinteris-Andreas:7/ANYmal_B_trained_using_SAC_on_gymnasium_mujoco-v5_framework:1?r=6fn5jA9uZQUZXGKVpwtqjz1eyJcS3hj3" allowfullscreen></iframe>
+ 
+ ### Example [Unitree Go1](https://github.com/deepmind/mujoco_menagerie/blob/main/unitree_go1/README.md)
+```py
+env = gymnasium.make('Ant-v5', xml_file='./mujoco_menagerie/unitree_go1/scene.xml', healthy_z_range=(0.195, 0.75), ctrl_cost_weight=0.05)
+```
+
+<iframe id="odysee-iframe" width="560" height="315" src="https://odysee.com/$/embed/@Kallinteris-Andreas:7/Unitree_Go1_trained_using_SAC_on_gymnasium_mujoco-v5_framework:5?r=6fn5jA9uZQUZXGKVpwtqjz1eyJcS3hj3" allowfullscreen></iframe>
```
Comment (Kallinteris-Andreas) [inline _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments:89]: @mgoulao do we want to host video on farama.org, can you handle the bandwidth for video playback

Comment (pseudo-rnd-thoughts) [reply _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments:89]: My thought is yes

File: _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments
Comment (pseudo-rnd-thoughts) [inline _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments:1]: Are spaces allowed in the filenames?

File: _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments (hunk 1 above)
Comment (mgoulao) [reply _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments:89]: Yes

File: _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments
Comment (mgoulao) [reply _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments:1]: They are, but I think it is good practice to replace them with - or _

Comment (Kallinteris-Andreas) [reply _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments:1]: Where do i have to put the files and how do i embed them on the article

Comment (pseudo-rnd-thoughts) [reply _posts/"2024-0?-??-Gymnasium-MuJoCo-v5 Environments:1]: @Kallinteris-Andreas Can you convert them to gif and insert them as photos?

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Pull request merged**